Add Quote Box Component

brenda-rg opened the pull request:
![image](https://github.com/user-attachments/assets/f0460a07-1bdc-4d74-b19c-750aa12f9beb)

Added div tags for box shadow and textbox
Added p tags to display text for quote and name
Added calling the component in page.tsx (home)
Changed the bg color on page.tsx so the component would be visible

Link to Issue - #12

stanleylew5 commented on src/app/page.tsx:
can you make it such that the strings are fetched from a data file? I'm not sure if we'll use multiple quotes but that's a future problem

stanleylew5 commented on src/components/home/QuoteBox.tsx:
```diff
@@ -0,0 +1,18 @@
+          <p className="text-2xl text-csa-yellow-100">“</p>
+          <p className="text-md text-right text-black">{quote}</p>
+        </div>
+        <p className="px-[2vw] text-left">- {name}</p>
```
```suggestion
        <p className="pl-[2vw]">- {name}</p>
```

stanleylew5 commented on src/components/home/QuoteBox.tsx:
make sure the entire component uses the lora font

stanleylew5 commented on src/components/home/QuoteBox.tsx:
```diff
@@ -0,0 +1,18 @@
+      <div className="absolute right-4 top-4 h-full w-full rounded-xl bg-csa-gray-200 bg-opacity-50"></div>
+      <div className="relative left-0 top-0 flex flex-col gap-8 rounded-xl bg-white p-4">
+        <div className="gap-4 px-[2vw] pt-2">
+          <p className="text-2xl text-csa-yellow-100">“</p>
```
increase the size of the quotation mark

stanleylew5 commented on src/components/home/QuoteBox.tsx:
```diff
@@ -0,0 +1,18 @@
+  return (
+    <div className="relative w-3/5">
+      <div className="absolute right-4 top-4 h-full w-full rounded-xl bg-csa-gray-200 bg-opacity-50"></div>
+      <div className="relative left-0 top-0 flex flex-col gap-8 rounded-xl bg-white p-4">
```
```suggestion
      <div className="relative flex flex-col gap-8 rounded-xl bg-white p-4">
```
remove these they shouldn't do anything

stanleylew5 commented:
Make sure you link your issue here 
![image](https://github.com/user-attachments/assets/845e9990-c92c-4119-ac6b-98ada2290fd9)

brenda-rg commented:
![image](https://github.com/user-attachments/assets/ddf9323a-7c52-4507-abfb-22f76121050f)

Fixed font
Fixed quote size
Added fetch from data file
Added data file
Removed unnecessary code

stanleylew5 reviewed (approved):
LGTM
